src/routes/docs/builders/[name]/+page.svelte

```diff
@@ -12,11 +12,13 @@
 
 	export let data: PageData;
 
+	// eslint-disable-next-line no-undef
 	type Component = $$Generic<typeof SvelteComponent>;
 	$: component = data.doc.default as unknown as Component;
 	$: doc = data.doc.metadata;
 	$: snippets = data.snippets;
 	$: mainPreview = data.mainPreview as unknown as Component;
+	// eslint-disable-next-line svelte/no-reactive-literals, svelte/no-immutable-reactive-statements
```

Comment:
`viewCode` below is marked as reactive, but it doesn't react to any variables and is a simple boolean. Can be solved whenever.

Reply:
You can change it to a let, no problem!

Reply:
Updated it in [`adb2d53` (#443)](https://github.com/melt-ui/melt-ui/pull/443/commits/adb2d5316e38eb25d9378e6ecc271d582b2d46d3)